packages/contracts/contracts/ChugSplashRegistry.sol

```diff
@@ -69,13 +70,24 @@ contract ChugSplashRegistry {
     /**
      * @notice Address of the ProxyUpdater.
      */
-    address public immutable proxyUpdater;
+    // TODO: Add immutable keyword once this contract is not upgradeable anymore.
```

Comment:
Same note as with `ChugSplashManager`, we can just use a constructor and put the immutables in the implementation contract

Reply:
Same thing here, it seems like we can't assign immutable variables outside of the constructor.